landlock.cpp

```diff
@@ -248,17 +248,19 @@ EX int isRandland(eLand l) {
   return 0;
   }
 
+EX bool landStartsFires(eLand l) {
+  return (l == laPower || l == laVolcano || l == laDragon || l == laEFire || l == laRlyeh);

```

Comment:
As discussed in comments:
```suggestion
  return (l == laPower || l == laVolcano || l == laDryForest || l == laDragon || l == laEFire || l == laRlyeh);

```